spec/comment_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  before(:example) do
+    @user = User.new(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
+    @post = Post.new(author: @user, title: 'Hello', text: 'This is my first post')
+  end
+
+  subject { Comment.new(user: @user, post: @post) }
+
+  it 'name should be present' do
+    subject.user = nil
+    expect(subject).to_not be_valid
+  end
+
```

Comment:
- [ ] It would be better if we write tests for custom methods of our models, for example; in the comment model we are expecting to test the method `update_comment_counter`